aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -29,6 +29,7 @@ spec:
                 operator: NotIn
                 values:
                 - fargate
+      serviceAccountName: ebs-csi-controller-sa
```

Comment:
Why does the daemonset require controller SA? What's the error you see in the original issue: #518?